doc/man3/EVP_PKEY_ASN1_METHOD.pod

```diff
@@ -214,7 +214,7 @@ It's called by L<EVP_PKEY_bits(3)>.
 The param_decode() and param_encode() methods are called to decode /
 encode DER formatted parameters to / from B<pk>.
 They MUST return 0 on error, 1 on success.
-They're called by L<PEM_read_bio_Parameters(3)> and the B<file:>
+They're called by PEM_read_bio_Parameters() and the B<file:>
```

Comment:
no, because there is no mention (not in the synopsis and not in the text) of that function in the manpage.

Reply:
So *add it*

Reply:
It's a separate PR, I'd have to learn what the function does, etc.  This is about removing broken links, not adding docs.

Reply:
1.  Did you look at the file I mentioned? How hard can it be to duplicate what is already there?
2.  What stops you from making that other PR instead of unlinking here (and having to waste energy arguing with me)?

Reply:
I am sorry if I've done anything untoward that annoyed you.

Reply:
#10113